src/main/java/org/jabref/gui/openoffice/OOBibBase.java

```diff
@@ -1259,6 +1259,71 @@ public void combineCiteMarkers(List<BibDatabase> databases, OOBibStyle style)
         }
     }
 
+    /**
+     * Do the opposite of combineCiteMarkers.
+     * Combined markers are split, with a space inserted between.
+     */
+    public void unCombineCiteMarkers(List<BibDatabase> databases, OOBibStyle style)
+            throws IOException, WrappedTargetException, NoSuchElementException, IllegalArgumentException,
+            UndefinedCharacterFormatException, UnknownPropertyException, PropertyVetoException, CreationException,
+            BibEntryNotFoundException {
+        XNameAccess nameAccess = getReferenceMarks();
+        // TODO: doesn't work for citations in footnotes/tables
```

Comment:
Are you going to work on this todo?

Reply:
This one is not really mine, https://github.com/JabRef/jabref/blob/3dc2039b49cc91952d8b908e255320008bf6033d/src/main/java/org/jabref/gui/openoffice/OOBibBase.java#L1189, just copied with the code of combineCiteMarkers.
There is some code in getReferenceMarks concerning footnotes, but not tables.

I did insert some TODO notes, however.

Some of these are questions, like: https://github.com/antalk2/jabref/blob/ff1bc8933e3b05630633ef262e8b6f7928446ba9/src/main/java/org/jabref/gui/openoffice/OOBibBase.java#L184, where I am not sure
if the code is there for a purpose (e.g. to provoke an exception if some assumption fails) or just forgotten.